qiime2/sdk/tests/test_plugin_manager.py

```diff
@@ -60,6 +68,36 @@ def test_importable_types(self):
 
     # TODO: add tests for type/directory/transformer registrations
 
+    def test_importable_formats(self):
+        obs = self.pm.importable_formats
+        exp = {
+            'IntSequenceDirectoryFormat': IntSequenceDirectoryFormat,
```

Comment:
These values aren't quite right, they should be `FormatRecord` objects. The assertion below is only testing the keys (since you're turning `dict` into `set`, which only inserts the dict's keys into the set).